site/app/controllers/admin/ConfigurationController.php

```diff
@@ -150,6 +153,18 @@ public function updateConfiguration(): MultiResponse {
         ) {
             $entry = $entry === "true" ? true : false;
         }
+        elseif ($name === 'queue_enabled') {
+            $entry = $entry === "true" ? true : false;
+        }
+        elseif ($name === 'queue_contact_info') {
+            $entry = $entry === "true" ? true : false;
+        }
```

Comment:
I dont think you want to change lines 156-161 as they have nothing to do with polls